packages/blockchain/src/index.ts

```diff
@@ -470,28 +468,29 @@ export default class Blockchain implements BlockchainInterface {
 
     async.series(
       [
-        verify,
+        async.asyncify(async function() {
```

Comment:
oh nice 👍

Reply:
a full refactor should be done to simplify all these mixed `async` methods. But that's already planned :)